src/irmin-pack/inode.ml

```diff
@@ -627,24 +667,21 @@ struct
               (function
                 | None -> true
                 | Some t ->
-                    let t = get_target ~find t in
+                    let t = Ptr.target la t in
```

Comment:
you could extract a `target_of_ptr` function outside of the `check` (as you did for `find_value` for instance)

Reply:
You're right. I forgot about this one.

This pattern also occurs in many recursive functions (e.g. `Ptr.target la` in `Val_impl.add`). I could try to fix those too.